pkg/orchestrator/config/config.go

```diff
@@ -27,6 +27,7 @@ const (
 	processNS       = "process_config"
 	defaultEndpoint = "https://orchestrator.datadoghq.com"
 	maxMessageBatch = 100
+	maxMessageSize  = 50 * 1e6 // 50 MB
```

Comment:
why 50MB ?

Reply:
because it was the closest multiple of 10 for the maximum uncompressed payload size observed over the course of recent months